### DISCUSSION TITLE
New to this packet radio stuff, but not to computing nor radio

### DISCUSSION THREAD

**Discussion by iainmaoileoin** [mfncooper/paracon#3] [Q&A]
I have direwolf on a PI.  I want to test communicate with it to a old TNC-220 (pac-comm) before  going back on-air.  Frequency (2025-2225) etc and wiring between the PI and the TNC are believed correct.

But I cant make direwolf do any of the "cmd: con MM7XXX" type commands when I telnet to port 8001- It says "cmd:" but otherwise does not like me!

It loked like paracon was the solution to my problem, but when I get it installed I have a wee window with a blue line saying "<font color=blue>C</font>onnect Disconnect                                      Not Connected
and a ">" prompt below that.

Anything I type to the ">" prompt just says "Not connected" in the status window.  

I am barking up the wrong tree?
Any advice appreciated.
Iain MM7IRM

**Comment by mfncooper** [comment]
The thing to understand here is that Direwolf, as a software TNC, *replaces* your PacComm hardware TNC. So the setup for the two systems looks roughly like this:

With hardware TNC:

    terminal emulator <--> PacComm TNC <--> radio

With software TNC:

    terminal application <--> Direwolf <--> sound card <--> radio

To communicate with your PacComm TNC, and work with commands at the 'cmd:' prompt, you'll want to use a terminal emulator (e.g. minicom) on your Pi that can connect via the serial port to your hardware TNC.

Direwolf does not implement the command interface of a hardware TNC. Instead, it provides software interfaces (e.g. AGWPE) that allow other application software to communicate with Direwolf and control its functionality.

Paracon communicates with Direwolf using the AGWPE software interface. When you connect to a remote station with Paracon, it is, conceptually, doing the same thing as you would if you typed 'con MM7XXX' when talking directly to your PacComm TNC using a terminal emulator. It's just that all the machinations are happening behind the scenes. You don't talk to the TNC, you talk to the application (Paracon) which talks to the TNC (Direwolf).

Finally, the reason you are seeing the "Not connected" message in Paracon is because, when you want to connect to a remote station with Paracon, you need to use the Connect command to tell Paracon what you want to do. See:

https://paracon.readthedocs.io/en/latest/userguide/#connections

You won't be typing commands directly to a TNC, as you would if you were using your PacComm TNC; instead, you tell Paracon what you want to do, and it will talk to Direwolf to make the connection.

I hope that helps!

**Reply by iainmaoileoin** [reply]
I believe I follow what you are saying.  I did not make myself clear.

I have a old/repaired TNC-220 that appears deaf.  I am trying to use direwolf (and some directly connected wires) to test my TNC-220.   I do know that the TNC-220 generates packets OK - I can run commands like CONNECT MM7IRM and it generates tones that the direwolf picks up and decodes.  So my pac-comm transmits and direwolf can listen to the packets.

I am trying to get direwolf (or ANYTHING!) to TX packets that I can have my TNC receive.

I have very little (none) activity around here on packet radio.  We are trying to get something running.  I am keen to use ol/original hardware to start with - if I can.  Later in life I may discard the old-slow-hardware TNC, but for now it is a project!

I now understand "https://paracon.readthedocs.io/en/latest/userguide/#connections"  I was typing C MM7XXX and just getting and error.  I am on the road now!

So, as they say RTFM!

Thanks

**Reply by iainmaoileoin** [reply]
That made all the difference!  Thanks.
Iain
MM7IRM

**Reply by mfncooper** [reply]
Ah, I see what you're doing now. Glad you got it sorted out!

**Comment by iainmaoileoin** [comment]
That made all the difference.  A nice piece of software.  Thanks.
Iain
MM7IRM
